dev/io.openliberty.persistence.3.2.thirdparty/resources/org/eclipse/persistence/internal/localization/i18n/ExceptionLocalizationResource.nlsprops

```diff
@@ -125,6 +137,8 @@ entity_manager_properties_conflict_default_connector_vs_external_transaction_con
 
 entity_manager_properties_conflict_default_connector_vs_jndi_connector=EntityManager properties conflict: jakarta.persistence.driver and/or jakarta.persistence.url require DefaultConnector, but jakarta.persistence.jtaDataSource and/or jakarta.persistence.nonjtaDataSource require JNDIConnector.
 
+entity_manager_with_connection_failed=Execution of user code failed: {0}
```

Comment:
Try not to use "execution." Maybe change it to:
The user code failed with an error: {0}